docs/locale/ja/LC_MESSAGES/faq.po

```diff
@@ -2,404 +2,449 @@
 msgid ""
 "A coroutine is a function that must be invoked with ``await`` or ``yield "
 "from``. When Python encounters an ``await`` it stops the function's "
 "execution at that point and works on other things until it comes back to "
 "that point and finishes off its work. This allows for your program to be "
-"doing multiple things at the same time without using threads or "
-"complicated multiprocessing."
+"doing multiple things at the same time without using threads or complicated "
+"multiprocessing."
 msgstr ""
+"コルーチンとは ``await`` または ``yield from`` から呼び出さなければならない関数です。 ``await`` "
+"にエンカウントした場合、そのポイントで関数の実行を停止し、他の作業を実行します。 これは作業が終了し、このポイントに戻ってくるまで続きます。 "
+"これにより、スレッドや複雑なマルチプロセッシングを用いずに複数の処理を並列実行することができます。"
 
 #: ../../faq.rst:26
 msgid ""
-"**If you forget to await a coroutine then the coroutine will not run. "
-"Never forget to await a coroutine.**"
-msgstr ""
+"**If you forget to await a coroutine then the coroutine will not run. Never "
+"forget to await a coroutine.**"
+msgstr "**コルーチンにawaitを記述し忘れた場合、コルーチンは実行されません。awaitの記述を忘れないように注意してください。**"
 
 #: ../../faq.rst:29
 msgid "Where can I use ``await``\\?"
-msgstr ""
+msgstr "``await`` はどこで使用することができますか。"
 
 #: ../../faq.rst:31
 msgid ""
-"You can only use ``await`` inside ``async def`` functions and nowhere "
-"else."
-msgstr ""
+"You can only use ``await`` inside ``async def`` functions and nowhere else."
+msgstr "``await`` は ``async def`` 関数の中でのみ使用できます。"
 
 #: ../../faq.rst:34
 msgid "What does \"blocking\" mean?"
-msgstr ""
+msgstr "「ブロッキング」とはなんですか。"
 
 #: ../../faq.rst:36
 msgid ""
-"In asynchronous programming a blocking call is essentially all the parts "
-"of the function that are not ``await``. Do not despair however, because "
-"not all forms of blocking are bad! Using blocking calls is inevitable, "
-"but you must work to make sure that you don't excessively block "
-"functions. Remember, if you block for too long then your bot will freeze "
-"since it has not stopped the function's execution at that point to do "
-"other things."
+"In asynchronous programming a blocking call is essentially all the parts of "
+"the function that are not ``await``. Do not despair however, because not all"
+" forms of blocking are bad! Using blocking calls is inevitable, but you must"
+" work to make sure that you don't excessively block functions. Remember, if "
+"you block for too long then your bot will freeze since it has not stopped "
+"the function's execution at that point to do other things."
 msgstr ""
+"非同期プログラミングにおけるブロッキングとは、関数内の ``await`` 修飾子がないコードすべてを指します。 "
+"しかし、全てのブロッキングが悪いというわけではありません。ブロッキングを使用することは避けられませんが、ブロックの発生は出来るだけ少なくする必要があります。長時間のブロックが発生すると、関数の実行が停止しないため、長時間Botがフリーズすることになることを覚えておきましょう。"
 
 #: ../../faq.rst:41
 msgid ""
 "A common source of blocking for too long is something like "
 ":func:`time.sleep`. Don't do that. Use :func:`asyncio.sleep` instead. "
 "Similar to this example: ::"
 msgstr ""
+"長時間ブロックの原因として一般的なのは ``time.sleep`` などです。 これは使用せず、下記の例のように ``asyncio.sleep`` "
```

Comment:
Reference directives should be maintained whereever possible as it creates links back to the Python documentation:
```diff
- 長時間ブロックの原因として一般的なのは ``time.sleep`` などです。 これは使用せず、下記の例のように ``asyncio.sleep``
+ 長時間ブロックの原因として一般的なのは :func:`time.sleep` などです。 これは使用せず、下記の例のように :func:`asyncio.sleep`
```
A comparison:
![image](https://user-images.githubusercontent.com/8049998/43718342-56d846e6-99c5-11e8-9ec5-064e5c346e79.png)
![image](https://user-images.githubusercontent.com/8049998/43718354-5c79e91a-99c5-11e8-8c66-47ef2283c544.png)